lib/game_classes/Board.js

```diff
@@ -1,78 +1,80 @@
-var Unoccupied = require('./Unoccupied.js');
+const Unoccupied = require('./Unoccupied');
 
-var Board = function (lengthOfSide) {
-    if (lengthOfSide < 0 || !Number.isInteger(lengthOfSide)) {
-        throw new ReferenceError('Instantiate board with a positive integer only.');
+class Board {
+    constructor (lengthOfSide) {
+        if (lengthOfSide < 0 || !Number.isInteger(lengthOfSide)) {
+            throw new ReferenceError('Instantiate board with a positive integer only.');
+        }
+        this.tiles = [];
+        this.lengthOfSide = lengthOfSide;
+        this.initializeBoard();
     }
-    this.tiles = [];
-    this.lengthOfSide = lengthOfSide;
-    this.initializeBoard();
-};
 
-Board.prototype.inspect = function () {
-    var horizontalDivide = '|';
-    for (var i=0; i<this.lengthOfSide; i++) {
-        var line = '|';
-        for (var j=0; j<this.lengthOfSide; j++) {
-            line += this.tiles[i][j].getCode() + '|';
+    inspect () {
+        var horizontalDivide = '|';
+        for (var i = 0; i < this.lengthOfSide; i++) {
```

Comment:
Please update all `var` to be `let`.

Reply:
@Asuza : Sure.. I mistakenly raised PR. Will make another commit in some time.

Reply:
Oh, sorry. Just let me know when you're done. 😄